frontend/src/components/shared/Radio/Radio.tsx

```diff
@@ -81,7 +81,7 @@ class Radio extends React.PureComponent<Props, State> {
 
     return (
       <div className="row-widget stRadio" style={style}>
-        <WidgetLabel label={label}>
+        <WidgetLabel label={label} disabled={isDisabled}>
```

Comment:
nit: Wouldn't it be possible here to just use the `disabled` variable instead of initializing the `isDisabled` (-> remove `isDisabled`)?  This seems to be something already added to the codebase, so maybe not relevant to refactor that in this PR.

Reply:
Removing `isDisabled` makes sense to me, so might as well refactor in this PR 👍🏼